examples/next-openai/app/stream-object/actions.ts

```diff
@@ -0,0 +1,28 @@
+'use server';
```

Comment:
```suggestion

```
This is unnecessary if there's `use server` in the function.

Reply:
thank you!

Reply:
put it back in, see comment about broken example